content/homeserver/pricing.md

```diff
@@ -47,6 +47,20 @@ Please find the answers to some questions below. For anything else please refer
 
 Yes, an account on the Free usage plan can upgrade a public room using the [room upgrade API](https://spec.matrix.org/latest/client-server-api/#post_matrixclientv3roomsroomidupgrade).
 
+### What types of data count towards the fair usage limits?
```

Comment:
Is it also worth explaining *why*  we have fair usage limits and what we're trying to prevent? i.e. we want to allow free exchanges of media as part of normal chat in rooms, but we do not support using matrix as a "file store".

Reply:
Good question. I made https://github.com/matrix-org/matrix.org/issues/2864 to track.